src/pages/Fox/components/FoxToken.tsx

```diff
@@ -0,0 +1,45 @@
+import type { StackProps } from '@chakra-ui/react'
+import { Box, Heading } from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { useTranslate } from 'react-polyglot'
+import { AssetIcon } from 'components/AssetIcon'
+import { FeeExplainer } from 'components/FeeExplainer/FeeExplainer'
+
+import { FoxTokenBalances } from './FoxTokenBalances'
+import { FoxTokenHeader } from './FoxTokenHeader'
+
+const stackProps: StackProps = {
+  flexDir: 'row',
+  flexWrap: { base: 'wrap', md: 'nowrap' },
+  my: 4,
+  maxWidth: '100%',
+  borderRadius: '10',
+  overflow: 'hidden',
+}
+const containerPaddingX = { base: 4, xl: 0 }
+
+export const FoxToken = () => {
+  const translate = useTranslate()
+
+  return (
+    <Box py={4} px={containerPaddingX}>
+      <Heading as='h2' fontSize='2xl' display='flex' alignItems='center'>
+        <AssetIcon assetId={foxAssetId} showNetworkIcon={false} me={2} />
+        {translate('foxPage.foxToken')}
```

Comment:
preferably-blocking: `assetId` looks programmatic, but is actually hardcoded in https://github.com/shapeshift/web/blob/32d187a5124b6fef851e1b82897764b7ee948386/src/pages/Fox/FoxPage.tsx#L17 and confirmed static by this copy, do we want to remove `assetId` as input/output from the FOX Page context ?